### DISCUSSION TITLE
Rename PDF file while Previewing

### DISCUSSION THREAD

**Discussion by seanawdish** [ccseer/Seer#416] [General]
The only thing I like better about Quicklook is that it will allow me to rename a PDF file while previewing the PDF. but Seer does not allow this and I get the error 'File in Use'. 

Is there an option I am missing in the settings to allow renaming while previewing?
If  isn't already available to allow this, can you PLEASE add support for this because Seer is a better overall preview tool than quicklook but this capability is the only thing keeping me from using it as my default.

I am using Windows 11.  Pro user, Version 3.2.1 U

**Comment by ccseer** [comment]
not possible. Seer allows you to preview a PDF which has 10000 pages or
sized more than 1GB. no way the software can read the whole file into
memory, so the file is always in use.

**Comment by ccseer** [comment]
sorry
fixed in 4.1.0
